doc/developer_guide/api.rst

```diff
@@ -51,13 +51,13 @@ User Interface
 .. automodapi:: glue.qt.widgets.data_viewer
    :no-inheritance-diagram:
 
-.. automodapi:: glue.qt.widgets.scatter_widget
```

Comment:
I wanted to include ``widget`` to show it's a Qt widget, and ``viewer`` as opposed to the options widget